spec/features/editing_images/upload_lead_image_spec.rb

```diff
@@ -32,7 +32,7 @@ def and_i_crop_the_image
     asset_manager_delete_asset(@asset_id)
     stub_publishing_api_put_content(Document.last.content_id, {})
     click_on "Crop image"
-    WebMock.reset!
+    WebMock::RequestRegistry.instance.reset!
```

Comment:
This looks a lot safer. Is it a private API? Should we add a todo comment to update this?

Reply:
Can do - an upcoming release of WebMock should expose it as a proper method :-).